dm/tests/duplicate_event/run.sh

```diff
@@ -73,11 +73,22 @@ function run_with_prepared_source_config() {
 	run_sql_source2 "show master status;"
 	binlog_file=$(grep "File" $TEST_DIR/sql_res.$TEST_NAME.txt | awk -F: '{print $2}' | xargs)
 	binlog_pos=$(grep "Position" $TEST_DIR/sql_res.$TEST_NAME.txt | awk -F: '{print $2}' | xargs)
-
 	server_uuid=$(tail -n 1 $WORK_DIR/worker2/relay-dir/server-uuid.index)
-	relay_log_size=$(ls -al $WORK_DIR/worker2/relay-dir/$server_uuid/$binlog_file | awk '{print $5}')
-	echo "binlog_pos: $binlog_pos relay_log_size: $relay_log_size"
-	[ "$binlog_pos" -eq "$relay_log_size" ]
+
+	succ=0
```

Comment:
wait the task `synced` in `query-status`?

Reply:
I just hope write XIDEvent will not exceed 5 seconds 😂

Reply:
waiting the syncer synced might be plausible.

Reply:
they will be also implemented as "check something and retry for a peroid". `synced` needs a DDL to trigger flush and needs to wait longer time.